lib/manager/gradle-wrapper/artifacts.ts

```diff
@@ -0,0 +1,82 @@
+import Git from 'simple-git/promise';
+import { resolve } from 'path';
+import * as fs from 'fs-extra';
+import { logger } from '../../logger';
+import { UpdateArtifact, UpdateArtifactsResult } from '../common';
+import { exec, ExecOptions } from '../../util/exec';
+import { readLocalFile } from '../../util/fs';
+import { platform } from '../../platform';
+import { VERSION_REGEX } from './search';
+import { gradleWrapperFileName, prepareGradleCommand } from '../gradle/index';
+
+async function addIfUpdated(
+  status: Git.StatusResult,
+  filePath: string
+): Promise<UpdateArtifactsResult | null> {
+  if (status.modified.includes(filePath)) {
+    return {
+      artifactError: null,
+      file: {
+        name: filePath,
+        contents: await readLocalFile(filePath),
+      },
+    };
+  }
+  return null;
+}
+
+export async function updateArtifacts({
+  packageFileName,
+  updatedDeps,
+  newPackageFileContent,
+  config,
+}: UpdateArtifact): Promise<UpdateArtifactsResult[] | null> {
+  try {
+    const projectDir = resolve(packageFileName, './../../../');
+    logger.debug(updatedDeps, 'gradle-wrapper.updateArtifacts()');
+    const gradlewPath = resolve(
+      projectDir,
+      `./${gradleWrapperFileName(config)}`
+    );
+    const version = VERSION_REGEX.exec(newPackageFileContent).groups.version;
+    prepareGradleCommand(
+      gradleWrapperFileName(config),
+      projectDir,
+      await fs.stat(gradlewPath).catch(() => null),
+      null
+    );
+    const cmd = `${gradlewPath} wrapper --gradle-version ${version} --project-dir ${projectDir}`;
```

Comment:
```suggestion
    const cmd = `${gradlewPath} wrapper --gradle-version ${version}`;
```

Reply:
`--project-dir ${projectDir}` must be present. Otherwise, gradle updates (or adds) gradle wrapper execution files to the root git repo folder. Try to remove this parameter locally and you'll see that.